zkevm-circuits/src/evm_circuit/witness.rs

```diff
@@ -1427,5 +1424,15 @@ pub fn block_convert(
                     })
             })
             .collect(),
+        copy_events: block
+            .copy_events
+            .iter()
+            .map(|copy_event| {
+                let key = Word::from(copy_event.tx_id)
```

Comment:
A tiny suggestion: maybe the key can be a tuple, so that accesses to the map become simple:
```rust
let key = (copy_event.tx_id, copy_event.call_id, copy_event.pc.0);
```